### PR TITLE
First commit for front

### PR DESCRIPTION
**Pull request by fabmel1**
Added the following:

1. Icon
2. Logo
3. Index.html created by React
4. Manifiest.json
5. Dockerfile

### CODE REVIEW

**Comment by candrepa1** [comment]
I think we can do without the logos

**Comment by fabmel1** [comment]
Logos removed. Please check

File: front/public/favicon.ico
Comment (candrepa1) [inline front/public/favicon.ico:1]: i'm not sure but we might also be able to remove this icon

File: front/public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (candrepa1) [inline front/public/index.html]: need to remove the links to the icons on this file

File: front/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
```
Comment (candrepa1) [inline front/public/manifest.json]: we can also remove them from here

File: front/src/App.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
```
Comment (candrepa1) [inline front/src/App.css]: we could check if these classes are being used anywhere. If not, we can remove them

**Comment by fabmel1** [comment]
Hi Camila,

All references to logos and icons, were removed. Thanks for the observation

**Review by candrepa1 — APPROVED** [review]

**Pull request merged**